### PR TITLE
replaced webcomponents and added SDK usage

### PR DESCRIPTION
**Pull request by alexbalakirev**

### CODE REVIEW

File: templates/profile.html [hunk 1]
```diff
@@ -5,31 +5,32 @@
...
-    const logoutButton = document.getElementById('logoutButton');
-
-    // Add event listener to logout button
-    logoutButton.addEventListener('click', function() {
-        session.logout()
-            .then(() => {
+    <h2>:/protected 🔒</h2>
+    <p>User ID: {{USER_ID}}</p>
+    <p>Name: {{USER_NAME}}"</p>
+    <p>Email: {{USER_EMAIL}}</p>
```
Comment (vincentdelitz) [inline templates/profile.html:16]: Can you add the passkey-list component here please (https://docs.corbado.com/frontend-integration/ui-components/passkeylist)

File: templates/profile.html [hunk 2]
```diff
@@ -5,31 +5,32 @@
...
+        <!-- Import Corbado SDK -->
+        <script src="https://{{ PROJECT_ID }}.frontendapi.corbado.io/auth.js"></script>
+        <script src="https://{{ PROJECT_ID }}.frontendapi.corbado.io/utility.js"></script>
+        <!-- Logout button -->
+        <button id="logoutButton">Logout</button>
+      
+        <script>
+          // Create a new session
+          const session = new Corbado.Session('{{ PROJECT_ID }}');
```
Comment (vincentdelitz) [inline templates/profile.html]: Session management in JavaScript has changed also significantly. Can you please take a look at the session wizard and try to get the right code (e.g. there's no "Corbado.Session" object in this sense anymore, you just need the Global Corbado object which also automatically refreshes the session.

**Review by vincentdelitz — COMMENTED** [review]
- [x] When I run `docker comopose up` I receive the following error:
image

- [x] Moreover, can you please add the passkey list component
- [x] Can you also take a look at the security warnings here in GitHub. I think most of them just require updating some of the packages.
- [x] Security warnings check -> most of them should be fixed by updating the packages.

**Comment by vincentdelitz** [comment]
Kriege nach dem Login einen 500er Internal Server Error derzeit:
image

File: templates/profile.html (hunk 2 above)
Comment (alexbalakirev) [reply templates/profile.html]: done

File: templates/profile.html (hunk 1 above)
Comment (alexbalakirev) [reply templates/profile.html:16]: done

**Pull request merged**